.github/workflows/os-osd-deployment.yml

```diff
@@ -61,6 +62,7 @@ jobs:
       kube-config: ${{ secrets.KUBE_CONFIG_DATA_PROD }}
       kibanaserver: ${{ secrets.KIBANASERVER }}
       ga-tracking-id: ${{ secrets.GA_TRACKING_ID_PROD }}
+      query_datasources_master_key: ${{ secrets.QUERY_DATASOURCES_MASTER_KEY }}
```

Comment:
can we change this in the dev and observability environment first, make sure that everything works fine, then touch Prod. thanks

Reply:
Ok.